httpevents/request.go

```diff
@@ -75,6 +75,9 @@ func (r *request) reset(req *http.Request, laddr string) {
 		raddr = "???"
 	}
 
+	// We delete the authorization header such that private secrets are not logged
+	req.Header.Del("Authorization")
```

Comment:
Is this on a copy of the request or the original? If it's the original - it would prevent the application from reading this header as well?

Reply:
I think maybe we want to delete from the copy instead:

```
r.reqHeaders.set(req.Header)
r.reqHeaders.Del("Authorization")
```

Reply:
ah good point. that would not be fun